examples/formats/json/python/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6.2
```

Comment:
Does it need it's own Dockerfile?  Looks like it's doing the basics that `fn build` already covers.